Classes/Console/Command/Extension/ExtensionActivateCommand.php

```diff
@@ -37,7 +37,7 @@ protected function configure()
         $this->addArgument(
```

Comment:
Here a relation to `extension:deactivate` and vice-versa would make sense.

Reply:
yeah. I'm happy to merge PRs which add command relations after this one is merged :)